__tests__/app.test.js

```diff
@@ -0,0 +1,22 @@
+const db = require('../db/connection.js');
+const app = require('../db/app.js');
+const seed = require('../db/seeds/seed.js');
+const { articleData, commentData, topicData, userData } = require('../db/data/test-data/index.js');
+const request = require("supertest");
+
+beforeEach(() => seed({ articleData, commentData, topicData, userData }));
+
+afterAll(() => db.end());
+
+describe('/api/topics', () => {
+    test('GET 200: returns an array of objects, which have properties of slug and description', () => {
+        return request(app).get('/api/topics')
+        .expect(200)
+        .then((response) => {
+            const topic = response.body[0];
+            expect(topic).toHaveProperty('slug');
+            expect(topic).toHaveProperty('description');
```

Comment:
Rather than just testing the first element in the array I'd like to see some iteration and testing each element in the returned array. Remember when using a forEach though that you should check the length of the array also.